.github/workflows/golang-binary-test.yml

```diff
@@ -0,0 +1,76 @@
+name: Monorepo CI/CD
+
+on:
+  push:
+    paths-ignore:
+      - 'services/excluded/**/*'
```

Comment:
you need to either use paths-ignore or paths you cant use both you can refer to :  https://github.com/persys-dev/persys-cloud/blob/main/.github/workflows/docker-image.yml and use the same path ignore basically.